malariagen_data/anoph/dipclust.py

```diff
@@ -23,14 +23,16 @@
     cnv_params,
 )
 from .snp_frq import AnophelesSnpFrequencyAnalysis
-from .cnv_data import AnophelesCnvData
+from .cnv_frq import AnophelesCnvFrequencyAnalysis
```

Comment:
Just wondering, why is this change here in this PR?

Reply:
`gene_cnv` used to be in `anopheles.py` and the code for `plot_diplotype_clustering_advanced` contains a comment saying that it needs to be moved. This has been done and it is now in `cnv_frq.py`. To be able to use it as `self.gene_cnv()`, the class `AnophelesDipClustAnalysis` thus has to inherit the functions of `AnophelesCNVFrequencyAnalysis`. It worked previously because the API classes (`Ag3` and `Af1` inherit this function) but it breaks the listing and coverage tests to have a class call functions that are not part of its inheritance tree. There were no tests until now so it wasn't raising an issue but that was hardly the best solution.

It may have been clearer to create a separate issue and PR to clean up `AnophelesDipClustAnalysis` and create the tests but given that the changes are relatively minor I thought it would be OK to do both at the same time.